File: hard/7_task_assignment/task_assignment.test.js
const  taskAssignment = require('./task_assignment');
// const assert = require('assert')

test("Test Case #1", () => {
  const k = 3;
  const tasks = [1, 3, 5, 3, 1, 4];
  const expected = [
    [4, 2],
    [0, 5],
    [3, 1],
  ];
  const actual = taskAssignment(k, tasks);
  expect(actual).toEqual(expected);
});

File: hard/7_task_assignment/task_assignment_test.py
import unittest
from task_assignment import taskAssignment

class TestProgram(unittest.TestCase):
  def test_case_1(self):
    k = 3
    tasks = [1, 3, 5, 3, 1, 4]
    expected = [
      [4, 2],
      [0, 5],
      [3, 1],
    ]
    actual = taskAssignment(k, tasks)
    self.assertEqual(actual, expected)
